src/model/advertisement.ts

```diff
@@ -106,7 +106,7 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'right',
             adTypes: ['mpu-banner-ad', 'rendered'],
             sizeMapping: {
-                mobile: ['1,1|2,2|300,250|300,274|300,600|fluid'],
+                mobile: ['1,1|2,2|300,250|300,274|fluid'],
                 // Pascal: It's not totally obvious where this sequence should come from the frontend code,
                 // but it should be correct.
```

Comment:
I think the use of [file marks][uuid] as below can be quite useful to navigate between `DCR` and `frontend`.

[uuid]: https://www.uuidgenerator.net/version4

Reply:
Ha, i'm not familiar with them but will have a look. It's coming from this file:

https://github.com/guardian/frontend/blob/beb2d69bfcb901403498d942291d59f62d2064dc/static/src/javascripts/projects/commercial/modules/article-aside-adverts.js#L70